src/main/java/com/yandex/money/api/typeadapters/BaseTypeAdapter.java

```diff
@@ -31,6 +31,7 @@
 
 import java.io.InputStream;
 import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
```

Comment:
You can't use this class because this library is also used by Android developers and in our SDK for Android. Class `StandardCharsets` was added in API 19, but acceptable minimum version is 15.